docker/Dockerfile.vppagent-firewall-nse

```diff
@@ -18,3 +18,6 @@ COPY --from=build /go/bin/vppagent-firewall-nse /bin/vppagent-firewall-nse
 RUN rm /opt/vpp-agent/dev/etcd.conf /opt/vpp-agent/dev/kafka.conf; echo 'Endpoint: "0.0.0.0:9112"' > /opt/vpp-agent/dev/grpc.conf; echo "disabled: true" > /opt/vpp-agent/dev/linux-plugin.conf
 COPY dataplane/vppagent/conf/vpp/startup.conf /etc/vpp/vpp.conf
 COPY examples/conf/vppagent-firewall-nse/supervisord.conf /etc/supervisord/supervisord.conf
+
+RUN rm -f /dev/shm/db /dev/shm/global_vm /dev/shm/vpe-api
```

Comment:
Can you explain why we need this step? Is it related to the issue?

Reply:
I've mimic the [CMD](https://github.com/ligato/vpp-agent/blob/485e388d53a46f6789fe0b0fca100f184cfbaee1/docker/dev/Dockerfile#L125) of the original image, which does this rm in addition to running supervisord

Reply:
see https://github.com/moby/moby/issues/5147